web/styles/popovers.css

```diff
@@ -56,6 +56,75 @@
     }
 }
 
+.tippy_popover {
```

Comment:
If you want to definite CSS specific to user card popover, then this is not a good classname.